DigitalLearningSolutions.Data.Migrations/Scripts/UAR-831-CreateViewsForAdminUsersAndCandidatesTables-UP.sql

```diff
@@ -0,0 +1,76 @@
+﻿CREATE VIEW AdminUsers AS
+SELECT dbo.AdminAccounts.AdminID,
+       null                                    AS Login,
+       dbo.Users.PasswordHash                  AS Password,
+       dbo.AdminAccounts.CentreID,
+       dbo.AdminAccounts.IsCentreAdmin         AS CentreAdmin,
+       0                                       AS ConfigAdmin,
+       dbo.AdminAccounts.IsReportsViewer       AS SummaryReports,
+       dbo.AdminAccounts.IsSuperAdmin          AS UserAdmin,
+       dbo.Users.FirstName                     AS Forename,
+       dbo.Users.LastName                      AS Surname,
+       dbo.Users.PrimaryEmail                  AS Email,
+       dbo.AdminAccounts.IsCentreManager,
+       1                                       AS Approved,
+       0                                       AS PasswordReminder,
+       ''                                      AS EITSProfile,
+       null                                    AS PasswordReminderHash,
+       null                                    AS PasswordReminderDate,
+       dbo.AdminAccounts.Active,
+       dbo.AdminAccounts.IsContentManager      AS ContentManager,
+       dbo.AdminAccounts.PublishToAll,
+       dbo.AdminAccounts.ImportOnly,
+       dbo.Users.TermsAgreed                   AS TCAgreed,
+       dbo.AdminAccounts.IsContentCreator      AS ContentCreator,
+       dbo.Users.FailedLoginCount,
+       dbo.Users.ProfileImage,
+       dbo.AdminAccounts.IsSupervisor          AS Supervisor,
+       dbo.AdminAccounts.IsTrainer             AS Trainer,
+       ISNULL(dbo.AdminAccounts.CategoryID, 0) AS CategoryID,
+       null                                    AS SkypeHandle,
+       0                                       AS PublicSkypeLink,
+       dbo.AdminAccounts.IsFrameworkDeveloper,
+       dbo.Users.ResetPasswordID,
+       dbo.AdminAccounts.IsFrameworkContributor,
+       dbo.AdminAccounts.IsWorkforceManager,
+       dbo.AdminAccounts.IsWorkforceContributor,
+       dbo.AdminAccounts.IsLocalWorkforceManager,
+       dbo.AdminAccounts.IsNominatedSupervisor AS NominatedSupervisor
+FROM dbo.Users
+         INNER JOIN dbo.AdminAccounts ON dbo.Users.ID = dbo.AdminAccounts.UserID
+GO
+
+CREATE VIEW Candidates AS
+SELECT dbo.DelegateAccounts.CandidateID,
+       dbo.DelegateAccounts.Active,
+       dbo.DelegateAccounts.CentreID,
+       dbo.Users.FirstName,
+       dbo.Users.LastName,
+       dbo.DelegateAccounts.DateRegistered,
+       dbo.DelegateAccounts.CandidateNumber,
+       dbo.Users.JobGroupID,
+       dbo.DelegateAccounts.Answer1,
+       dbo.DelegateAccounts.Answer2,
+       dbo.DelegateAccounts.Answer3,
+       null                          AS AliasID,
+       dbo.DelegateAccounts.Approved AS Approved,
+       dbo.Users.PrimaryEmail        AS EmailAddress,
+       dbo.DelegateAccounts.ExternalReg,
+       dbo.DelegateAccounts.SelfReg,
+       dbo.Users.PasswordHash        AS Password,

```

Comment:
With the change of `Password` to `OldPassword` on `DelegateAccounts`, should we be returning that here, or are we keeping the `Users.PasswordHash` here, and making new functions later on for the specific login case?

Reply:
We'll keep Users.PasswordHash here and make new functions later on.